### PR TITLE
Docsite: update user_guide/playbooks_module_defaults

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Docsite: update user_guide/playbooks_module_defaults

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/user_guide/playbooks_module_defaults.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

**Review by kuterahul515 — APPROVED** [review]

File: docs/docsite/rst/user_guide/playbooks_module_defaults.rst [hunk 1]
```diff
@@ -120,8 +132,12 @@ In a playbook, you can set module defaults for whole groups of modules, such as
         group/aws:
           region: us-west-2
       tasks:
-      - aws_s3_bucket_info:
+      - name: Get info
+        community.aws.aws_s3_bucket_info:
```
Comment (s-hertel) [inline docs/docsite/rst/user_guide/playbooks_module_defaults.rst]: This doesn't work like that yet.

~Also note, while setting the action FQCN in module_defaults will let the task use it if it's called with the same FQCN, redirected names don't work yet. The module_defaults action has to be an exact match with how the module is called. IMO it seems a little premature to collectionify the documentation until those things work.~ 

Edit - Oh, I fixed that and forgot it was merged. Just the `groups/` change here and below is not right.

Comment (Andersson007) [reply docs/docsite/rst/user_guide/playbooks_module_defaults.rst]: @s-hertel thanks for the feedback! I'll take a look at this tomorrow

**Comment by Andersson007** [comment]
cc @samccann @acozine

**Comment by acozine** [comment]
Thanks for adding the `name: ` lines and making things consistent @Andersson007.

**Pull request merged**